### PR TITLE
pekwm: update to 0.1.18

### PR DESCRIPTION
**Pull request by biopsin**
Switched to cmake (notified about depricated issue https://github.com/pekdon/pekwm/issues/28)
Switched distfiles to pekdon@github
Become maintainer
Heads up : jpeg support will be dropped in next release
Tested : armv7l, i686, x86_84/-musl

### CODE REVIEW

File: srcpkgs/pekwm/template [hunk 1]
```diff
@@ -1,33 +1,17 @@
...
-	/etc/pekwm/mouse
-	/etc/pekwm/menu
-	/etc/pekwm/start
-	/etc/pekwm/keys
-	/etc/pekwm/autoproperties_typerules
-	/etc/pekwm/config
-	/etc/pekwm/autoproperties
-	/etc/pekwm/mouse_system
-	/etc/pekwm/mouse_sloppy
-	/etc/pekwm/mouse_click
```
Comment (ericonr) [inline srcpkgs/pekwm/template:32]: Did it stop using these files?

Comment (biopsin) [reply srcpkgs/pekwm/template:32]: No they are still in use. 
Hm, ok I see what the conf_files purpose is (pkg ownership),
I shall correct my blunder in a short while.

**Comment by ericonr** [comment]
Could you please rebase this?

**Comment by biopsin** [comment]
oh crap..

**Comment by biopsin** [comment]
Ok, I'm closing this PR as I have made a complete mess out of my branch.
Also noticed makefile -DSYSCONFDIR points to /usr/etc/pekwm and not /etc/pekwm
it fails initializing on startup.

**Pull request closed without merging**